ui_framework/components/color_picker/_color_picker.scss

```diff
@@ -13,12 +12,27 @@
     height: $colorPickerSize;
     border-radius: $globalBorderRadius;
     box-shadow: inset 0 0 0 1px rgba(#000, 0.2);
+    display: inline-block;
+  }
+
+  .kuiColorPicker__emptySwatch {
+    svg {
+      position: absolute;
+      width: $colorPickerSize;
+      height: $colorPickerSize;
+    }
+
+    svg line {
+      stroke: rgb(255, 0, 0);
+      stroke-width: 2;
+    }
   }
 
   .kuiColorPicker__label {
     font-size: $globalFontSize;
     line-height: $globalLineHeight;
     margin-left: 10px;
+    margin-right: 10px;
```

Comment:
If we use my suggestion to use `KuiFieldGroup` to lay out the form elements, then we can remove this `margin-right`.